app/pages/signup/signup.js

```diff
@@ -168,6 +169,14 @@ export let Signup = translate()(class extends React.Component {
         options.loginHint = inviteEmail;
       }
       keycloak.register(options);
+
+      if (ReactGA.isInitialized) {
+        ReactGA.event({
+          category: 'redirect',
+          action: 'Keycloak Sign Up',
+          label: inviteEmail ? 'Has inviteEmail' : 'No inviteEmail',
+        });
+      }
```

Comment:
The `keycloak.register()` call above this is ultimately a redirect, so I'm concerned that this will end up being a race condition as to whether or not this event gets sent properly. Given the async nature of _both_ of the calls, it might not help a ton, but could be worth moving this above the `keycloak.register()`